src/pages/Dashboard/sub-pages/IssueRequests/IssueRequestsTable/index.tsx

```diff
@@ -29,7 +29,7 @@ import { QUERY_PARAMETERS } from 'utils/constants/links';
 import { TABLE_PAGE_LIMIT } from 'utils/constants/general';
 import graphqlFetcher, { GraphqlReturn, GRAPHQL_FETCHER } from 'services/fetchers/graphql-fetcher';
 import genericFetcher, { GENERIC_FETCHER } from 'services/fetchers/generic-fetcher';
-import issueFetcher, { ISSUE_FETCHER, getIssueWithStatus } from 'services/fetchers/issue-request-fetcher';
+import issuesFetcher, { ISSUES_FETCHER, getIssueWithStatus } from 'services/fetchers/issues-fetcher';
```

Comment:
For consistency, I updated the names. I found that variables and files were not properly named.